tests/comments.test.js

```diff
@@ -0,0 +1,186 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+let accessToken= null;
+let idComment= null;
+
+describe("GET Tests Comments ID discussion",() =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
describe("GET Tests Comments ID discussion",() => {
```